go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/mattermost/mmctl
 
-go 1.13
+go 1.14
```

Comment:
Wondering why is this needed? Unless we are raising the minimum required Go version to compile mmctl (which I don't think we are), there's no need to change this.

Reply:
Right! Changed back to `1.13`